src/main/java/net/kodehawa/mantarobot/core/command/TextContext.java

```diff
@@ -506,9 +576,78 @@ public UtilsContext getUtilsContext() {
         return new UtilsContext(getGuild(), getMember(), getChannel(), i18n, null);
     }
 
+    public List<Member> getMentionedMembers() {
+        var members = getEvent().getMessage().getMentions().getMembers();
+        if (isMentionPrefix) {
+            var mutable = new LinkedList<>(members);
+            return mutable.subList(1, mutable.size());
+        }
+
+        return members;
+    }
+
+    public List<User> getMentionedUsers() {
+        var users = getEvent().getMessage().getMentions().getUsers();
+        if (isMentionPrefix) {
+            var mutable = new LinkedList<>(users);
+            return mutable.subList(1, mutable.size());
+        }
+
+        return users;
+    }
 
     @Override
     public I18nContext getLanguageContext() {
         return i18n;
     }
+
+    public void setLanguageContext(I18nContext languageContext) {
+        this.i18n = languageContext;
+    }
+
+    public I18nContext getGuildLanguageContext() {
+        return new I18nContext(getDBGuild(), null);
+    }
+
+    public Color getMemberColor(Member member) {
+        return member.getColor() == null ? Color.PINK : member.getColor();
+    }
+
+    public Color getMemberColor() {
+        return getMemberColor(getMember());
+    }
+
+    // Both used for options.
+    public void setCustomContent(String str) {
+        this.customContent = str;
+    }
+
+    /**
+     * Get the custom (usually filtered) content. This is only used in options, do not call it anywhere else.
+     * @return The custom content that has been set using Context#setCustomContent
+     */
+    public String getCustomContent() {
+        return this.customContent;
+    }
+
+    /**
+     * Get all the content in a message.
+     * @return all the content in a message, including new lines, or empty if none.
+     */
+    public String getContent() {
+         var content = tryArgument(Parsers.remainingContent());
+        return content.map(String::trim).orElse("");
+    }
+
+    public boolean isChannelNSFW() {
+        if (getChannel() instanceof IAgeRestrictedChannel txtChannel) {
+            return txtChannel.isNSFW();
+        }
+
+        return true;
+    }
```

Comment:
Same as the other 2 comments about default in IContext. This should probably also not default to true?